docassemble/MotionToStayEviction/data/questions/review_screen.yml

```
@@ -0,0 +1,386 @@
+---
+event: review_case_info
+id: |-
+  review screen
+question: |-
+  Review your answers
+review:
+  - Edit: users.revisit
+    button: |
+      **Your party: (Edit to change name and address)**
+
+      %  for my_var in users:
+        * ${ my_var.name }
+      % endfor
+  - Edit: other_parties.revisit
+    button: |
+      **The other party: (Edit to change name and address)**
+      
+      % for my_var in other_parties:
+        * ${ my_var.name }
+      % endfor
+  - Edit: users[0].address.address
+    button: |
+      **Your address:**
+      ${ users[0].address.on_on_line() }
+  - Edit: |-
+      case_new_or_existing
+    button: |
+      **Is this a new case?**
+
+      ${ showifdef('case_new_or_existing') }
+  - Edit: |-
+      users.there_is_another
+    button: |
+      **Did anyone else file with you?**
+
+      Is anyone else on your side of the case?: ${ word(yesno(users.there_is_another)) }
+  - Edit: |-
+      public_assistance_kinds
+    button: |
+      **Do you get any of these benefits?**
+
+      ${ public_assistance_kinds.true_values() }
+  - Edit: |-
+      has_household_members
+    button: |
+      **About your income and household**
+
+      Does anyone else live with you as part of your household?: ${ word(yesno(has_household_members)) }
+
+      How many people? Don't count yourself.: ${ showifdef('household_additional_size') }
+
+      What is the total (after tax) income for your household?: ${ showifdef('hh_income.value') }
+  - Edit: |-
+      can_afford
+    button: |
+      **Can you afford to pay the court costs?**
+
+      I can afford the costs: ${ word(yesno(can_afford)) }
+  - Edit: |-
+      fees["Filing fee"].waive
+    button: |
+      **What fees do you need the court to waive or pay?**
+
+      Filing fee or e-filing fee: ${ word(yesno(fees["Filing fee"].waive)) }
+
+      Amount (if known): ${ showifdef('fees["Filing fee"].amount') }
```

```suggestion
      % if defined('fees["Filing fee"].amount'):
      Amount: ${ fees["Filing fee"].amount }
      % endif
```
For this and the others, this review section is really big for the amount of content most people will have.